src/main/java/org/folio/circulation/domain/ItemStatus.java

```diff
@@ -54,4 +54,8 @@ void setDate(String date) {
   private boolean valueMatches(String value) {
     return equalsIgnoreCase(getValue(), value);
   }
+
+  public boolean isLost() {
+    return this == DECLARED_LOST || this == AGED_TO_LOST;
+  }
```

Comment:
I thought the other check (in the refund context) also included lost and paid, is this a different rule?

Reply:
See this comment https://github.com/folio-org/mod-circulation/pull/655#discussion_r483613499